lib/arbitrary.js

```diff
@@ -150,6 +150,67 @@ function oneof() {
   });
 }
 
+// Return a lazy arbitrary that delegates to another arbitrary at its
+// 'strict' property. An arbitrary must be assigned to that property before
+// this arbitrary can generate anything.
+function lazyArbitrary() {
+  var arb = {};
+  // This function must be pure because it will not be called with
+  // meaningful context.
+  arb.generator = generator.bless(function (size) {
+    return arb.strict.generator(size);
+  });
+  arb.shrink = shrink.noop;
+  arb.show = show.def;
+  arb = arbitraryBless(arb);
+  return arb;
+}
+
+/**
+  - ```js
+    letrec(
```

Comment:
This has been fixed in the latest version. I don't know how to get Git to show that in the PR conversation, but you can see it in the [commit](https://github.com/jsverify/jsverify/pull/193/commits/ef44826cdc738763af3ee22f886154300091fd72).